.circleci/config.yml

```diff
@@ -166,6 +166,8 @@ jobs:
             -PreleaseKeyAlias=$RELEASE_KEY_ALIAS \
             -PreleaseKeystorePassword=$RELEASE_KEYSTORE_PASSWORD \
             -PreleaseKeyPassword=$RELEASE_KEY_PASSWORD
+      - store_artifacts:
+          path: purchases-sample/build/outputs/apk/debug/purchases-sample-debug.apk
```

Comment:
this works correctly and can be installed, but after doing so it just gets stuck on a loading screen. @vegaro do you have any thoughts on what I might be missing here? This is the same app that we use for manual testing, right?